docs/UserGuide.adoc

```diff
@@ -179,14 +179,26 @@ After entering an attribute, the system will prompt the user input for another a
 
 Format: `add t/**TYPE**`
 
-ORGANice will prompt the user to type in each attribute, one by one.
+When the user type in the command above, a form will be displayed. The form should looks like this:
 
-In the display box, there will be a form that reflects the attributes typed by the user.
+image::form.png[width="790"]
 
-After the inputs are added, ORGANice will display a template that shows what the user data will look like
-if the changes are finalised. Press the Enter key to confirm the changes.
+ORGANice will prompt the user to type in each attribute, one by one until all the fields are filled.
```

Comment:
Add in an example after this sentence. The purpose is to walk the user through each step. 
A suggestion: "Firstly, the display box will display a message, prompting the user to enter the patient's name. To add a patient that is named "Tommy", type in `n/Tommy` in the command box and press Enter. After entering, the patient's name will be displayed in the form, as shown in Figure... (attach a screenshot also). 
This process will be continued until all the fields are filled. (attach a screenshot to show a filled list)"